File: app/server_php/classes/business/DisruptedService.php
<?php

/**
 * Description of DisruptedService
 */
class DisruptedService {

  public $from;
  public $to;
  public $detailPath;
  public $description;
  
  public $stoppingPoints = array();

  public function __toString() {
    return $this->from . " -> " . $this->to . " : " . $this->detailPath;
  }

}

File: app/api_go/business/disruptedservice.go
package business

import (
  "fmt"
)

type DisruptedService struct {
  From            string              `json="from"`
  To              string              `json="to"`
  Description     string              `json="description"`
  StoppingPoints  map[string]string   `json="stoppingPoints"`
}

func (service DisruptedService) String() string {
  return fmt.Sprintf("%24s -> %-24s : %s : %v", service.From, service.To, service.Description, service.StoppingPoints)
}
